src/main/java/com/woowahan/techcourse/cart/ui/MemberArgumentResolver.java

```diff
@@ -24,7 +25,8 @@ public boolean supportsParameter(MethodParameter parameter) {
     }
 
     @Override
-    public Object resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer, NativeWebRequest webRequest, WebDataBinderFactory binderFactory) throws Exception {
+    public Object resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer,
+            NativeWebRequest webRequest, WebDataBinderFactory binderFactory) throws Exception {
```

Comment:
```suggestion
    public Member resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer,
            NativeWebRequest webRequest, WebDataBinderFactory binderFactory) {
```
overridng 했지만 명시적으로 변경해도 좋을것 같네요!

Reply:
아하 감사합니다! 👍